dash.el

```diff
@@ -357,7 +357,7 @@ See also: `-each-indexed'."
        (nreverse ,r))))
 
 (defun -map-when (pred rep list)
-  "Return a new list where the elements in LIST that does not match the PRED function
+  "Return a new list where the elements in LIST that do not match the PRED function
```

Comment:
Actually, I confused this function with `-remove` :D The original is fine, sorry for the confusion.

(Though according to the emacs doc conventions, first line should be <80 char complete sentence... I don't see any way to describe this in such a short sentence :O)